### ISSUE TITLE
unique() throws depending on generator

### ISSUE THREAD

**Issue by nicolaiskogheim** [fzaninotto/Faker#856]
This works as expected:
```php
$faker->unique()->words($nb = 3, $asText = true)
```

This does not:
```php
$faker->unique()->words($nb = 3, $asText = false)
```

Stacktrace from the latter:
```
  [ErrorException]                                                                
  array_key_exists(): The first argument should be either a string or an integer  


Exception trace:
 () at /data/vendor/fzaninotto/faker/src/Faker/UniqueGenerator.php:44
 Illuminate\Foundation\Bootstrap\HandleExceptions->handleError() at n/a:n/a
 array_key_exists() at /data/vendor/fzaninotto/faker/src/Faker/UniqueGenerator.php:44
 ```

It may be that I just don't understand how `unique` works, but I was surprised that I got this error.

**Comment by fzaninotto** [comment]
This was already fixed in #749, please test the master branch.

**Comment by nicolaiskogheim** [comment]
Sorry, I should've done that. Thanks.
